### PR TITLE
(helm-charts): Changing port for supporting non-root in kubera-core-ui

### PR DESCRIPTION
**Pull request by rajdas98**
- Changing port for supporting non-root in kubera-core-ui

Ref- https://github.com/mayadata-io/kubera-core-ui/pull/57

### CODE REVIEW

File: kubera-enterprise/templates/kubera-core-ui-service.yaml [hunk 1]
```diff
@@ -6,6 +6,6 @@ spec:
   ports:
     - name: http
       port: 9091
-      targetPort: 80
+      targetPort: 8080
```
Comment (avishnu) [inline kubera-enterprise/templates/kubera-core-ui-service.yaml:9]: Why is this change?

Comment (rajdas98) [reply kubera-enterprise/templates/kubera-core-ui-service.yaml:9]: We are moving `kubera-core-ui` image from `nginx` to `unprivileged nginx` and bydefault unprivileged nginx listens on 8080
ref- https://github.com/mayadata-io/kubera-core-ui/pull/57

Comment (avishnu) [reply kubera-enterprise/templates/kubera-core-ui-service.yaml:9]: Got it

**Review by avishnu — APPROVED** [review]
lgtm

**Pull request merged**